core-services/importExport/src/main/java/org/silverpeas/core/importexport/coordinates/CoordinatesPositionsType.java

```diff
@@ -23,41 +23,38 @@
  */
 package org.silverpeas.core.importexport.coordinates;
 
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
 import java.util.List;
 
 /**
  * @author dlesimple
  */
+@XmlRootElement
+@XmlAccessorType(XmlAccessType.NONE)
 public class CoordinatesPositionsType {
 
+  @XmlElement(name = "coordinatePosition")
   private List coordinatesPositions;
+  @XmlAttribute
   private boolean createEnable = false;
 
-  /**
-   * @return List of CoordinatesPositions
-   */
+  public CoordinatesPositionsType() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1186)